transform/models/staging/synapse/stg_synapse__access_approval.sql

```diff
@@ -0,0 +1,21 @@
+with
+
+source as (
+    select * from {{ source('rds', 'access_approval') }}
+),
+
+staging as (
+    select
+        id as access_requirement_approval_id,
+        requirement_id as access_requirement_id,
+        requirement_version as access_requirement_version,
+        submitter_id,
```

Comment:
For discussion on overall data modeling in the future: should all references to Synapse User Profiles be suffixed with `user_id`?  like `submitter_user_id`?, `created_by_user_id`, etc?